File: scala/WesleySafadao.scala
case class Safadao(day: Int, month: Int, year: Int) {
  def saySafadeza() = {
    val saf  = (1 to month + 1).sum  + (year / 100.0) * (50 - day)
    val anjo = 100 - saf
    println(s"Você é $saf% safado e $anjo% anjo")
  }
}

File: swift/WesleySafadao.swift
func sum(v: Int) -> Int {
    return (0...v).reduce(0, +)
}

func safadao(day: Int, month: Int, year: Int) -> (anjo: Double, safado: Double) {
    let s = sum(v: month)

    let safado = Double(s) + (Double(year)/100) * (50 - Double(day))
    let anjo = 100 - safado

    return (anjo: anjo, safado: safado)
}
